### PR TITLE
Use n1-highcpu-32 machine and cache base image for 2 days in CI (#231)

### PR DESCRIPTION
**Pull request by jeremyje**
Use a larger machine so we can build more artifacts in parallel. Increase the Kaniko cache for the base CI image since it barely changes and the tooling does not change day-to-day.

### CODE REVIEW

**Comment by jeremyje** [comment]
> [Comments](https://github.com/GoogleCloudPlatform/open-match/blob/8dcdc460ece5300aa4ef0be60b39618c85cc94cf/cloudbuild.yaml#L202) in line 50 and line 202-207 are now resolved and should be deleted to increase file readability.

I've addressed these.

File: cloudbuild.yaml [hunk 1]
```diff
@@ -47,10 +47,9 @@
 #    https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/go
 
 steps:
-# Blocked by https://github.com/GoogleContainerTools/kaniko/issues/477
 - id: 'Docker Image: open-match-build'
   name: gcr.io/kaniko-project/executor
-  args: ['--destination=gcr.io/$PROJECT_ID/open-match-build', '--cache=true', '--cache-ttl=6h', '--dockerfile=Dockerfile.ci', '.']
+  args: ['--destination=gcr.io/$PROJECT_ID/open-match-build', '--cache=true', '--cache-ttl=48h', '--dockerfile=Dockerfile.ci', '.']
```
Comment (sawagh) [inline cloudbuild.yaml]: whats the motivation for this?

Comment (jeremyje) [reply cloudbuild.yaml]: Added in the PR description. The cache retention was too short. The builds are still taking a while and caching (while the benefit is marginal) is short so there's more rebuilds than necessary.

**Pull request merged**